builder/manywheel/build_wheel.sh

```diff
@@ -20,8 +18,7 @@ docker run --rm -it \
     --env PLAT_NAME="${PLAT_NAME}" \
     --env USERID="${USERID}" \
     --env GROUPID="${GROUPID}" \
-    --env GIT_BRANCH="${GIT_BRANCH}" \
-    --env GIT_REMOTE="${GIT_REMOTE}" \
+    --volume "$(pwd)/../../:/lmdeploy" \
```

Comment:
is `lmdeploy_root=$(pwd)/../lmdeploy` when the following step runs?
https://github.com/InternLM/lmdeploy/blob/450757b2bdf3b124cfa14950e4f8ed6f6d15550c/.github/workflows/pypi.yml#L34

Reply:
yes, `cd "$(dirname "$0")"` will move workdir to where the scripts exists. And the original command has this line `--volume "$(pwd)/entrypoint_build.sh:/entrypoint_build.sh" \`

Reply:
sorry. didn't see line 13